### PR TITLE
Allow Grafana watcher to monitor multiple directories

### PR DESCRIPTION
**Pull request by unguiculus**

### CODE REVIEW

File: contrib/grafana-watcher/main.go [hunk 1]
```diff
@@ -16,31 +16,67 @@ package main
...
+type watchDirSet map[string]struct{}
+
+func (w *watchDirSet) String() string {
+	s := *w
+	return strings.Join(s.asSlice(), ",")
+}
+
+func (w *watchDirSet) Set(value string) error {
+	s := *w
+	cols := strings.Split(value, ",")
```
Comment (brancz) [inline contrib/grafana-watcher/main.go]: I'm not a huge fan of introducing both `--flag=a --flag=b,c`, let's just stick with a repeatable flag `--flag=a --flag=b --flag=c`.

**Comment by brancz** [comment]
Thanks a lot for your contribution!

Could you also bump the usage of the v0.0.6 tag everywhere in this repo and make sure it still works as expected? Specifically the manifest in `contrib/kube-prometheus/`. I just want to make sure manifests are adapted if needed especially regarding the pflags switch.

File: contrib/grafana-watcher/main.go (hunk 1 above)
Comment (unguiculus) [inline contrib/grafana-watcher/main.go]: I just copied this from kube-state-metrics.

Comment (brancz) [inline contrib/grafana-watcher/main.go]: Interesting, I guess then it's better to be consistent. Thanks for pointing it out.

**Comment by brancz** [comment]
Already pushed the image to the quay repo. Nice work! lgtm

**Pull request merged**